docker/docker-compose.yml

```diff
@@ -1,5 +1,23 @@
 version: '3.3'
 services:
+  redis:
+    image: redis:${REDIS_VERSION:-3.2-alpine}
+    hostname: travis-redis
+    volumes:
+      - ../redis:/data
+    command: redis-server --appendonly yes
+    networks:
+      - resque
+
+  redis-cache:
+    image: redis:${REDIS_VERSION:-3.2-alpine}
+    hostname: travis-redis-cache
+    volumes:
+      - ../redis:/data
+    command: redis-server --save ""
+    networks:
+      - redis_cache
+
```

Comment:
As far as I can see, there is no change, do we need this cut and paste?  Is docker-compose.yml order dependent?

Reply:
Since some days, Redis is taking around 50 secondes to start in Travis, it's better for the stack to launch them first.